<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>500</height>
   </size>
  </property>
  <property name="font">
   <font>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>Définition  des mesures</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="demarrer">
    <property name="geometry">
     <rect>
      <x>451</x>
      <y>180</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Démarrer une série de mesure</string>
    </property>
    <property name="text">
     <string>Démarrer les mesures</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>2</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Temperature interne :</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>461</x>
      <y>260</y>
      <width>151</width>
      <height>192</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Temperatures</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="labelCam">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>391</width>
      <height>261</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>3</y>
      <width>261</width>
      <height>151</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QSpinBox" name="spinMini">
     <property name="geometry">
      <rect>
       <x>173</x>
       <y>10</y>
       <width>60</width>
       <height>27</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>850</number>
     </property>
     <property name="value">
      <number>40</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinMaxi">
     <property name="geometry">
      <rect>
       <x>173</x>
       <y>49</y>
       <width>60</width>
       <height>27</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>850</number>
     </property>
     <property name="value">
      <number>60</number>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="spinInter">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>62</width>
       <height>27</height>
      </rect>
     </property>
     <property name="minimum">
      <double>0.050000000000000</double>
     </property>
     <property name="value">
      <double>0.100000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Temperature maxi :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Temperature mini :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>92</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Intervalle :</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>3</y>
      <width>241</width>
      <height>111</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="labelTempInt">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>131</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>441</x>
      <y>170</y>
      <width>191</width>
      <height>301</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>40</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>actuelle :</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>111</width>
      <height>131</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="labelConsigne">
     <property name="geometry">
      <rect>
       <x>17</x>
       <y>67</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Consigne</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Quitter</string>
    </property>
   </widget>
   <zorder>frame_3</zorder>
   <zorder>frame_4</zorder>
   <zorder>frame</zorder>
   <zorder>frame_2</zorder>
   <zorder>demarrer</zorder>
   <zorder>label_5</zorder>
   <zorder>tableWidget</zorder>
   <zorder>labelCam</zorder>
   <zorder>label_7</zorder>
   <zorder>pushButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConfigurer_des_mesures">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Configuration</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionConfiguration_avanc_e"/>
   </widget>
   <addaction name="menuConfigurer_des_mesures"/>
  </widget>
  <action name="actionConfiguration_avanc_e">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Configuration avancée</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>demarrer</tabstop>
  <tabstop>tableWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
